### PR TITLE
I think this fixes the path issues

### LINKED ISSUE #24: Couldn't setup offworld
Trying to setup the offworld WP site, we ran into a problem running `setup-script.py`. Using this command:

`python setup-script.py -n offworld -u offworldevents.com -t "Offworld Events" -o owkube`

We got this error:

```
Traceback (most recent call last):
  File "setup-script.py", line 215, in <module>
    main()
  File "setup-script.py", line 205, in main
    generate_manifests(template_vars, args.out)
  File "setup-script.py", line 171, in generate_manifests
    dest_file = dest / file_or_dir.relative_to(K8_DIR)
  File "/Users/colindavey/.pyenv/versions/3.7.3/lib/python3.7/pathlib.py", line 876, in relative_to
    .format(str(self), str(formatted)))
ValueError: '/Users/colindavey/Documents/MVP/standard-wordpress/k8s/routing.tmpl.yml' does not start with 'k8s'
```

### PR DESCRIPTION
**Pull request by oliverdain**
@colindavey please verify this works by pulling this branch and then re-running the command you have in #24 but with `--dry_run` given so it doesn't actually push anything to the cluster.

fixes #24

### CODE REVIEW

**Comment by colindavey** [comment]
Still getting an error, but a different one: 
```
$ python setup-script.py -n offworld -u offworldevents.com -t "Offworld Events" -o owkube --dry_run
2021-04-07 10:39:25,220 Setting project name to offworldevents-com
2021-04-07 10:39:25,221 Looking for Kubernetes files and templates in /Users/colindavey/Documents/MVP/standard-wordpress/k8s
2021-04-07 10:39:25,221 Expanding template /Users/colindavey/Documents/MVP/standard-wordpress/k8s/routing.tmpl.yml to owkube/owkube/routing.yml
Traceback (most recent call last):
  File "setup-script.py", line 215, in <module>
    main()
  File "setup-script.py", line 205, in main
    generate_manifests(template_vars, args.out)
  File "setup-script.py", line 180, in generate_manifests
    with open(dest_file, 'wt', encoding='utf-8') as outf:
FileNotFoundError: [Errno 2] No such file or directory: 'owkube/owkube/routing.yml'
```

File: setup-script.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def generate_manifests(template_vars: Dict[str, str], dest: Path) -> None:
             if file_or_dir.is_dir():
                 to_explore.append(file_or_dir)
             else:
-                dest_file = dest / file_or_dir.relative_to(K8_DIR)
+                dest_file = dest / file_or_dir.absolute().relative_to(K8_DIR.absolute())
                 dest_file.parent.mkdir(parents=True, exist_ok=True)
                 if str(file_or_dir).endswith(TEMPLATE_SUFFIX):
                     dest_file = dest_file.parent / (str(dest_file)[:-len(TEMPLATE_SUFFIX)] + '.yml')
```
Comment (colindavey) [inline setup-script.py]: I had luck changing this line to 
```dest_file = (str(dest_file)[:-len(TEMPLATE_SUFFIX)] + '.yml')```

Comment (oliverdain) [reply setup-script.py]: You've found the right bug, but I think that fix doesn't work if `dest_file` isn't a relative path. I added a similar fix that I think works with absolute and relative paths.

**Comment by oliverdain** [comment]
OK, _now_ I think it's fixed. Please try it again @colindavey .

**Comment by colindavey** [comment]
Looks good! 
```
$ python setup-script.py -n offworld -u offworldevents.com -t "Offworld Events" -o owkube --dry_run
2021-04-08 17:57:41,539 Setting project name to offworldevents-com
2021-04-08 17:57:41,539 Looking for Kubernetes files and templates in /Users/colindavey/Documents/MVP/standard-wordpress/k8s
2021-04-08 17:57:41,539 Expanding template /Users/colindavey/Documents/MVP/standard-wordpress/k8s/routing.tmpl.yml to owkube/routing.yml
2021-04-08 17:57:41,571 Looking for Kubernetes files and templates in /Users/colindavey/Documents/MVP/standard-wordpress/k8s/maintenance
2021-04-08 17:57:41,571 Expanding template /Users/colindavey/Documents/MVP/standard-wordpress/k8s/maintenance/phpmyadmin.tmpl.yml to owkube/maintenance/phpmyadmin.yml
2021-04-08 17:57:41,590 Looking for Kubernetes files and templates in /Users/colindavey/Documents/MVP/standard-wordpress/k8s/running
2021-04-08 17:57:41,591 Expanding template /Users/colindavey/Documents/MVP/standard-wordpress/k8s/running/mariadb.tmpl.yml to owkube/running/mariadb.yml
2021-04-08 17:57:41,614 Expanding template /Users/colindavey/Documents/MVP/standard-wordpress/k8s/running/wordpress.tmpl.yml to owkube/running/wordpress.yml
```

**Review by colindavey — APPROVED** [review]

**Comment by oliverdain** [comment]
@colindavey I'm going to assume this fixed the issue. If you have a chance to double-check and let me know I can re-open this if the issue isn't resolved.

**Pull request merged**